cli/pkg/kctrl/cmd/app/release/app_spec_builder.go

```diff
@@ -39,14 +41,17 @@ func NewAppSpecBuilder(depsFactory cmdcore.DepsFactory, logger logger.Logger, ui
 const (
 	LockOutputFolder = ".imgpkg"
 	LockOutputFile   = "images.yml"
+
+	inMemoryAppName      = "kctrl-builder"
```

Comment:
I thought I would need these constants while checking for errors again, but apparently the "run after" function receives a copy of the app, so it is not needed.

Reply:
So I am happy to remove the new constants if y'all think it is unnecessary